docs/source/index.rst

```diff
@@ -1,73 +1,84 @@
 .. raw:: html
 
-   <div style="width: 100%; text-align: center;">
-     <iframe src="_static/logo.html" style="width: 575px; height: 220px; border: none; margin: 0; padding: 0;" />
+   <div style="width: 100%; text-align: left; margin-left:-10px">
+     <iframe src="_static/logo.html" style="width: 555px; height: 155px; border: none; margin: 0; padding: 0;" />
       [SMaHT submitr logo]
      </iframe>
    </div>
 
-.. warning::
-
-   **THIS IS A PRE-RELEASE VERSION.**
-   Version 1.0 is forthcoming circa February 2024.
-
+-----------------------------
 
 ========
 Overview
 ========
 
-------------------------------------------------
-smaht-submitr: A remote file submitter for SMaHT
-------------------------------------------------
+**smaht-submitr**: A metadata and file submission tool for `SMaHT Portal <https://data.smaht.org/>`_.
 
-.. image:: https://github.com/smaht-dac/submitr/actions/workflows/main.yml/badge.svg
-    :target: https://github.com/smaht-dac/submitr/actions
-    :alt: Build Status
+.. .. image:: https://github.com/smaht-dac/submitr/actions/workflows/main.yml/badge.svg
+..     :target: https://github.com/smaht-dac/submitr/actions
+..     :alt: Build Status
 
-.. image:: https://coveralls.io/repos/github/smaht-dac/submitr/badge.svg
-    :target: https://coveralls.io/github/smaht-dac/submitr
-    :alt: Coverage Percentage
+.. .. image:: https://coveralls.io/repos/github/smaht-dac/submitr/badge.svg
+..     :target: https://coveralls.io/github/smaht-dac/submitr
+..     :alt: Coverage Percentage
 
-.. image:: https://readthedocs.org/projects/submitr/badge/?version=latest
-    :target: https://submitr.readthedocs.io/en/latest/?badge=latest
-    :alt: Documentation Status
+.. .. image:: https://readthedocs.org/projects/submitr/badge/?version=latest
+..     :target: https://submitr.readthedocs.io/en/latest/?badge=latest
+..     :alt: Documentation Status
```

Comment:
Are these intended? Look like comment outs and cause below to render as commented out as well? Maybe Github issue?

Reply:
I commented these out - seemed like unuseful to show these little gifs indicating build/etc status.